api/src/main/java/org/openmrs/module/m2sysbiometrics/util/M2SysProperties.java

```diff
@@ -0,0 +1,26 @@
+package org.openmrs.module.m2sysbiometrics.util;
+
+import org.apache.commons.lang.StringUtils;
+import org.openmrs.api.APIException;
+import org.openmrs.api.AdministrationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class M2SysProperties {
```

Comment:
I meant for this methods like: getCustomerKey(), getLocalBioServerUrl() etc. so that other classes wouldn't need to deal with it